### PR TITLE
Drop calico info logs

### PR DESCRIPTION
**Pull request by SamClinckspoor**
Drop INFO level logs from calico
https://github.com/skyscrapers/engineering/issues/107

### CODE REVIEW

File: templates/helm-values-fluentd-cloudwatch.tpl.yaml [hunk 1]
```diff
@@ -141,7 +141,7 @@ fluentdConfig: |
     </parse>
   </source>
 
-  # Drop kubeapi trace,debug and info logs
+  # Drop kubeapi info logs
```
Comment (duboisph) [inline templates/helm-values-fluentd-cloudwatch.tpl.yaml]: If limiting to dropping info, shouldn't you update the pattern bellow accordingly too?

File: templates/helm-values-fluentd-cloudwatch.tpl.yaml [hunk 2]
```diff
@@ -150,12 +150,12 @@ fluentdConfig: |
...
 
-  # Drop calico trace,debug and info logs
-  <filter kubernetes.var.log.containers.kube-apiserver-**>
+  # Drop calico info logs
+  <filter kubernetes.var.log.containers.calico-node-**>
     @type grep
     <exclude>
       key log
-      pattern pattern \[(TRACE|DEBUG|INFO)\]
+      pattern pattern \[INFO\]
```
Comment (duboisph) [inline templates/helm-values-fluentd-cloudwatch.tpl.yaml]: There's a `pattern` too much :)

File: templates/helm-values-fluentd-cloudwatch.tpl.yaml (hunk 1 above)
Comment (SamClinckspoor) [reply templates/helm-values-fluentd-cloudwatch.tpl.yaml]: It was already the case, just updated the comment to reflect it.

File: templates/helm-values-fluentd-cloudwatch.tpl.yaml (hunk 2 above)
Comment (SamClinckspoor) [reply templates/helm-values-fluentd-cloudwatch.tpl.yaml]: fixed

File: templates/helm-values-fluentd-cloudwatch.tpl.yaml (hunk 1 above)
Comment (SamClinckspoor) [reply templates/helm-values-fluentd-cloudwatch.tpl.yaml]: ah wait sorry, one sec

**Comment by SamClinckspoor** [comment]
@iuriaranda I disagree. Trace and debug logs are actually useful if something is wrong with the services.

**Comment by duboisph** [comment]
Aren't the verbosity levels as following: TRACE < DEBUG < INFO < WARN < ERROR. If so, I agree with @iuriaranda that it doesn't make much sense to keep debug/trace level logs while filtering info

**Comment by iuriaranda** [comment]
@SamClinckspoor in case something goes wrong, `debug` and `trace` logs won't tell you much without the `info` logs. 
Afaik you normally select the verbosity of the logs, from `trace` to `critical`, so if you set the threshold in `info`, you'll get `info`, `warning`, `error` and `critical` logs. If you set it in `debug`, you'll get everything except `trace`.

`trace` and `debug` will normally be much more verbose than `info`, even when everything is running correctly, so if we drop `info` logs to safe disk space, it doesn't make much sense to keep `trace` and `debug`. In any case, if something goes wrong, we'll need to increase the verbosity to debug the problem.

**Comment by SamClinckspoor** [comment]
changed again to drop all 3.
fyi, if you have a problem you will probably not have the info you need if its not exported.

**Comment by iuriaranda** [comment]
I agree, but afaik that was the compromise we accepted when we decided to safe disk space by lowering the log verbosity of system components.

**Pull request merged**